go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/sashabaranov/go-openai v1.36.1
```

Comment:
I don't understand why you need to update go.mod file

Reply:
returned it